src/assignments/scrabble-scorer.rst

```diff
@@ -0,0 +1,157 @@
+Assignment #2: Scrabble Scorer
+==============================
+
+To see how far you skills have come, we are going to give you a mission to update
+our *super* important Scrabble Scorer program. What, did you think you were going
+to work on Mars rover code already?
+
+The current Scrabble Scorer program will take in a word and return the Scrabble value for
```

Comment:
I think we should give some justification for WHY we need to redo the scoring algorithm. Is there a problem with having the points as the keys? Do we want to use different points for certain letters? Will the new algorithm boost efficiency?

If we want to make the assignment sound more realistic, then we should provide a reason why the change needs to be made.

Reply:
What do you think of this? "The current Scrabble Scorer, is a program that will take in a word and return the point value.
The program needs to be rewritten for two reasons. First the data structure used to store
the letter point values needs to be replaced. Second the program should have three scoring
algorithms and allow the user to pick which algorithm to use. These new features will make
the program more efficient and user friendly."

Reply:
This sounds good, and I like the "new features" approach.  However, there is still a question about WHY the data structure needs to be replaced ("First..."). The second reason might be enough justification on its own.